docs/5___ssd.adoc

```diff
@@ -530,6 +556,10 @@ Note that source and destination in the following table indicate the resulting d
 Implementations *MUST NOT* specify connections that are not of one of the allowed combinations in the following table.
 Implementations *MUST* ensure that data flow is specified unambiguously, including ensuring that not multiple connections with inbound data flow enter into a connector signifying an input, inout, parameter, or structuralParameter connector of an element, or a local, constant, calculatedParameter, or output connector of an enclosing system.
 
+For connectors of kind unspecified it is ultimately implementation-defined whether and how connections are allowed, as the exact semantics depend on the underlying modeling language.
```

Comment:
highlight 'unspecified' as a technical term.

Reply:
We do not currently handle this fully consistently throughout the document, I think, however here the names are used without highlight, see non-changed sentence above. So if we wanted to change this, we would need to change this throughout, I think.